### ISSUE TITLE
[`fud2`]

### ISSUE THREAD

**Issue by ethanuppal** [calyxir/calyx#2354]
Should the [docs](https://docs.calyxir.org/running-calyx/fud2/index.html) be updated to reflect our new position on `fud2` on all relevant pages? This would include replacing as many examples of using `fud` with `fud2`.

**Comment by rachitnigam** [comment]
#2348 is a blocking issue for this IMO

**Comment by jku20** [comment]
I agree. #2348 is related to stuff I worked on and fixing it is on my radar. I’ve just not yet gotten around to it.

I started work on making lowering the rhai a bit less bad, but haven’t gotten around to actually finish anything. Hopefully I’ll can get it done this summer when I have a bit more free time.
